src/ToolBox/SOS/Strike/sildasm.cpp

```diff
@@ -65,6 +65,7 @@ static OpCode opcodes[] =
 
 static ULONG position = 0;
 static BYTE *pBuffer = NULL;
+static char* asString(CQuickBytes *out);
```

Comment:
Why was this move needed?

Reply:
It is defined an extern function in the header file today yet the definition in the source file is static. GCC warns as signature failure. I looked to see if the function is shared with other files. The answer was no. I ended up moving the definition inside the source file where it is needed.